app/api/v1/views/user.py

```diff
@@ -0,0 +1,174 @@
+    @requires_permission('store attendant')
+    @end_point.expect(username)
+    @end_point.doc('read all users')
+    @end_point.marshal_with(user_message , code=200)
+    def get(self, username):
+        if not user.get(username):
+            return {'message': 'Sorry this user is not found',
+                    'user': {}
+                    }, 404
+        return {'message': 'success',
+                'user': user.get(username)
+                }, 200
+
+    @requires_permission('store attendant')
+    @end_point.expect(username, a_user)
+    @end_point.doc('update specific user')
+    @end_point.marshal_with(user_message, code=200)
+    def put(self, username):
+        req = end_point.payload
+        if not user.get(username):
+            return {'message': 'Sorry this user is not found'}, 404
+        data = {
+            'username': req['username'],
+            'first_name': req['first_name'],
+            'last_name': req['last_name'],
+            'password': req['password'],
+            'email': req['email']
+        }
+
+        if user.update(username, data):
+            return {'message': 'success',
+                    'user': user.get(username)
+                    }, 200
+        else:
+            return{
+                'message': 'Sorry could not update this user'
+            }
+
+    @requires_permission('admin')
+    @end_point.expect(username)
+    @end_point.doc('delete specific user')
+    @end_point.marshal_with(message, code=200)
+    def delete(self, username):
+        if not user.get(username):
+            return {'message': 'Sorry this user is not found'}, 404
+        if user.delete(username):
+            return {'message': 'success'}, 200
+        else:
+            return{
+                'message': 'Sorry could not delete this user'
+            }, 205
+
+
+@end_point.route('login/')
+class Login(Resource):
+    @end_point.expect(credentials)
+    @end_point.doc('login a user')
+    def post(self):
+        data = end_point.payload
+        if not data:
+            return {"message": 'Could not sing in unknown user'}, 400
+        username = str.strip(data.get('username'))
+        password = str.strip(data.get('password'))
+        if not username and password:
+            return {"message": "Username or password missing"}, 206
+        if user.sign_in(username, password):
+            return {
+                'token': str(create_access_token(identity=username)),
+                'message': 'Login successful!'
+            }, 200
+        return {"message": 'Could not sing in user'}, 401
+
+
+@end_point.header('Authorization: Bearer', 'JWT TOKEN', required=True)
+@end_point.route('logout/')
+class Logout(Resource):
+    @requires_permission('store attendant')
+    def post(self):
+        json_token_id = get_raw_jwt()['jti']
+        blacklist.add(json_token_id)
+        if json_token_id in blacklist:
+            return {'message': 'Logged out successfully'}, 200
+        else:
```

Comment:
I don't think this line is not necessary, at what point a user can't be logged out?